enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -648,80 +686,29 @@ proceed across the set on a per-cluster basis and, optionally, in the order spec
-    or `Failed`. (The `TimeOut` status would not be reflected.)
-  - For convenience and discoverability, add a per-cluster status to the policy set (this could alternatively be a
-    constructed string for rollout status as the policy status message currently is).
-
-- **Snippet**
-
-  ```golang
-  // ClusterStatus defines a per-cluster status for the policy set
-  type ClusterStatus struct { // <-- New struct
-    ComplianceState  ComplianceState               `json:"compliant,omitempty"`
-    RolloutStatus    clusterv1alpha1.RolloutStatus `json:"rolloutStatus,omitempty"`
-    ClusterName      string                        `json:"clustername,omitempty"`
-  }
-
-  // PolicySetStatus defines the observed state of PolicySet
-  type PolicySetStatus struct {
-    Placement     []PolicySetStatusPlacement    `json:"placement,omitempty"`
-    Compliant     string                        `json:"compliant,omitempty"`
-    RolloutStatus clusterv1alpha1.RolloutStatus `json:"rolloutStatus,omitempty"` // <-- New field
-    ClusterStatus []*ClusterStatus              `json:"clusters,omitempty"`      // <-- New field
-    StatusMessage string                        `json:"statusMessage,omitempty"`
-  }
-  ```
+  - Add the `manualPerGroup` rollout strategy as an option in the `spec.rolloutStrategy.type` field. It accepts all the
+    same arguments as `progressivePerGroup`.
+  - Add the `spec` portion to `Rollout` CRD.
 
 ### Notes/Constraints
 
-For the `Placement` library, this requires importing at least this package version (the `Placement` library is in the
-`v1alpha1` version):
-
-```
-  open-cluster-management.io/api v0.11.1-0.20230828015110-b39eb9026c6e
-```
-
-For testing, the `governance-policy-propagator` doesn't currently account for multiple managed clusters. As part of this
-enhancement, the test flows would need to be enhanced (and/or a separate workflow created) that deploys multiple managed
-clusters.
-
 The `PlacementDecisionGetter` returns an array of pointers (`[]*clusterv1beta1.PlacementDecision`) because it was
 intended to retrieve from a cache, so the implementation could consider setting up a `PlacementDecision` cache instead
 of using the Kubernetes client directly.
 
 ### Risks and Mitigation
 
-- The `Placement` library is relatively new and untested outside of its repo and this implementation leans heavily on
-  its logic. While it works in theory, there could be some tweaks/adjustments as development proceeds, lengthening time
-  for implementation. The phased approach intends to address this to make partial implementation feasible.
+- The `Placement` library does not support the `manualPerGroup` rollout strategy.
 
 ## Design Details
 
 ### Open Questions
 
-1. Should the per-cluster status on the root policy be grouped similar to how they're grouped in the
-   `PlacementDecisions`?
+1. Should the `spec.rolloutStrategy.ignoreClusterRolloutStatus` field be contributed to the rollout strategy API?
```

Comment:
ignoreClusterRolloutStatus -> is this act like ignore cluster group failure ? in other words, it is not calculated in maxfailure?  correct my understand?

Reply:
Essentially, the cluster gets the new policy version when its group is being rolled out to, but its status is ignored and not counted towards failure or success.